File: pgy_parent/pyg_shop_web/src/main/webapp/js/service/tbGoodsService.js
app.service("tbGoodsService",function ($http) {

    /* 添加商品 */
    this.add = function (entity) {
        return $http.post("../goods/add",entity);
    }

    /* 搜索框 */
    this.sreachGoods = function (pageNum,pageSize,searchEntity) {
        return $http.post("../goods/sreachGoods/" + pageNum  + "/" + pageSize,searchEntity);
    };

    this.updateMarketable = function (selectIds,isMarketable) {
        return $http.get("../goods/updateMarketable/"+selectIds+"/"+isMarketable);
    }

    this.deleIsDele = function (selectIds,IsDeleid) {
        return $http.get("../goods/deleIsDele/"+selectIds+"/"+IsDeleid)

    }
});

File: pgy_parent/pyg_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/TbGoodsService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import com.pinyougou.pojo.TbGoods;
import domainGroup.Goods;
import domaincommon.PageResult;

import java.util.List;

/**
 * Create with www.dezhe.com
 *
 * @Author 德哲
 * @Date 2018/7/15 9:34
 */
public interface TbGoodsService {

    /**
     * 添加商品
     * @param goods
     */
    void getAddGoods(Goods goods);

    /**
     * 搜索品牌功能
     * @param pageNum
     * @param pageSize
     * @param tbGoods
     * @return
     */
    PageResult searchGoods(int pageNum, int pageSize, TbGoods tbGoods);


    void updateAuditStatus(Long[] selectIds, String auditStatus);
    void updateMarketable(Long[] selectIds, String isMarketable);

    void deleIsDele(Long[] selectIds, String isDeleid);

    Goods findOne(Long goodsId);

    List<Goods> findAllGoods();
}
